plugin.rb

```diff
@@ -57,6 +58,7 @@ class DiscourseEncrypt::Engine < Rails::Engine
     User.class_eval              { prepend UserExtensions }
     Email::Sender.class_eval     { prepend EmailSenderExtensions }
     UserNotifications.class_eval { prepend UserNotificationsExtensions }
+    Guardian.class_eval      { prepend GuardianExtensions }
```

Comment:
```suggestion
    Guardian.class_eval          { prepend GuardianExtensions }
```

Nit-pick: spacing here is a bit off.